integration-test/README.md

```diff
@@ -47,3 +51,32 @@ For dev environment:
 ```shell
 npm run test-dev
 ```
+
+#### Postman Collection import in repo
+
+Link - https://app.getpostman.com/join-team?invite_code=188096ad281000ecd33932a724d74e5f&target_code=d1271d0429db3e9126cc48e18bfa08fb
```

Comment:
this link is not valid, or I am missing something. How to important just access via browser? or should I do it inside of postman?

Reply:
yes, the link is no valid anymore. I`ll change it for new one without time limits.  New link - https://womencodingcommunity.postman.co/workspace/OrgPlatform~39b013e7-f18f-43a2-88f2-e78632be555d

Reply:
I have added new link to the file

Reply:
I would suggest to export postman collections and environments with .json extension and add them into repository directly instead of referring to postman workspace. It will be much easier for collaborators to download files and import to their workspaces. 
In this case you have to write instructions only for importing collections and envs.

Reply:
@nora-weisser could you have a look and approve if it is how you suggested?

Reply:
> I would suggest to export postman collections and environments with .json extension and add them into repository directly instead of referring to postman workspace. It will be much easier for collaborators to download files and import to their workspaces. In this case you have to write instructions only for importing collections and envs.

I agree, better to just place the .json collection file and replace everytime we are creating/updating endpoints